app/uk/gov/hmrc/homeofficesettledstatusstubs/services/StubDataService.scala

```diff
@@ -16,73 +16,90 @@
 
-  private def search[S <: Searchable](searchable: S)(firstCheck: S => Option[StatusCheckResult])(
-    makeString: String)(fallBack: S => Option[ErrorResponse]): Result =
+  private def search[S <: Searchable](
+    searchable: S
+  )(firstCheck: S => Option[StatusCheckResult])(makeString: String)(fallBack: S => Option[ErrorResponse]): Result =
     resultFor(searchable.correlationId, firstCheck(searchable))(makeString)(fallBack(searchable)) match {
-      case Some(response @ ErrorResponse(_, status, _)) => new Status(status)(Json.toJson(response))
+      case Some(response @ ErrorResponse(_, status, _))                                     => new Status(status)(Json.toJson(response))
       case Some(response @ SuccessResponse(_, result)) if searchable.validateResult(result) =>
         Ok(Json.toJson(response))
-      case _ =>
+      case _                                                                                =>
         NotFound(
           StatusResponse
-            .errorResponseBody(searchable.correlationId, "ERR_NOT_FOUND", NOT_FOUND, Nil))
+            .errorResponseBody(searchable.correlationId, "ERR_NOT_FOUND", NOT_FOUND, Nil)
+        )
     }
 
   def mrzSearch(mrzSearch: MrzSearch): Result =
-    search(mrzSearch)(s => StubData.mrzToResult.get((s.docType, s.documentNum)))(
-      mrzSearch.documentNum)(s => checkOtherMrz(s.correlationId, s.docType, s.documentNum))
+    search(mrzSearch)(s => StubData.mrzToResult.get((s.docType, s.documentNum)))(mrzSearch.documentNum)(s =>
+      checkOtherMrz(s.correlationId, s.docType, s.documentNum)
+    )
 
   def ninoSearch(ninoSearch: NinoSearch): Result =
     search(ninoSearch)(s => StubData.ninoToResult.get(s.nino))(ninoSearch.givenName)(s =>
-      checkOtherNinos(s.correlationId, s.nino))
+      checkOtherNinos(s.correlationId, s.nino)
+    )
 
   private def resultFor(correlationId: String, search: => Option[StatusCheckResult])(
-    makeString: String)(fallBack: => Option[ErrorResponse]): Option[StatusResponse] =
+    makeString: String
+  )(fallBack: => Option[ErrorResponse]): Option[StatusResponse] =
     search match {
       case Some(result) => Some(SuccessResponse(correlationId, result))
-      case None =>
-        if (makeString.toUpperCase.startsWith("MAKE"))
+      case None         =>
+        if (makeString.toUpperCase.startsWith("MAKE")) {
           Some(SuccessResponse(correlationId, makeResponse(makeString)))
-        else
+        } else {
           fallBack
+        }
     }
 
   private def makeResponse(makeString: String): StatusCheckResult = {
     val _ :: product :: status :: expired =
       makeString.toUpperCase.replace("--", "_").split("-").toList
 
-    val now = LocalDate.now()
-    val isExpired = expired.contains("EX")
+    val now         = LocalDate.now()
+    val isExpired   = expired.contains("EX")
     val expiredDate = Some(if (isExpired) now.minusDays(1) else now.plusDays(1))
     StatusCheckResult(
       "Michael Makson",
```

Comment:
For test data in the open - please follow this guidance : https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open

Reply:
done